src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml

```diff
@@ -0,0 +1,1194 @@
+<UserControl x:Class="CalculatorApp.GraphingNumPad"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:Windows10version1803="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract, 7)"
+             xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
```

Comment:
We should use the standard naming convention for these namespaces:
`contract7NotPresent`, `contract7Present`